CampaignAutomation/Models/RLTContentConverter.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  RLTContentConverter.swift
+//  RealifeTech
+//
+//  Created by YOU-HSUAN YU on 2022/6/10.
+//  Copyright © 2022 Realife Tech. All rights reserved.
+//
+
+import Foundation
+
+public struct RLTContentConverter {
+
+    public init() { }
+
+    public func convert(factories: [RLTContentType: RLTCreatableFactory], items: [RLTContentItem]) -> [RLTCreatable] {
+        var views: [RLTCreatable] = []
```

Comment:
Could also be (in pseudocode)

return items.map { item in
   guard let factory = ... else{ return nil }
   return factory.create(from: item.data)
}.compactMap { $0 }

Would be more functional programming-style

Reply:
Thank you for the suggestion! I've updated it.